.npmignore

```diff
@@ -0,0 +1 @@
+types/test/
```

Comment:
this is so the tests don't get packaged in the resulting npm package.

Reply:
Can you update `files` in `package.json` instead?

Reply:
Done in [fa198ce](https://github.com/mtth/avsc/pull/476/commits/fa198ce467728f9e37389799066b47f1829956a4) and 
8d0d137